src/main/java/org/broadinstitute/hellbender/tools/exome/acsconversion/ACSModeledSegment.java

```diff
@@ -0,0 +1,69 @@
+package org.broadinstitute.hellbender.tools.exome.acsconversion;
+
+import org.broadinstitute.hellbender.tools.exome.ModeledSegment;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+
+/**
+ * Class to represent AllelicCapSeg (R version) segments.  This class should only be used for conversion operations.
+ */
+public class ACSModeledSegment extends ModeledSegment {
+
+    final double f;
+    final double tau;
+    final double sigmaTau;
+    final double muMinor;
+    final double sigmaMinor;
+    final double muMajor;
+    final double sigmaMajor;
+    final int segLabelCNLOH;
+    final int hetCount;
+
+    public ACSModeledSegment(final SimpleInterval interval, final String call, final long targetCount, final double segmentMeanInLogCR, final int hetCount, final double f, final double sigmaTau, final double muMinor, final double sigmaMinor, final double muMajor, final double sigmaMajor, final int segLabelCNLOH) {
```

Comment:
segmentMeanInLog2CR to be absolutely clear.  Actually, please do a find+replace on "InLogCR", it appears in a few other places.